src/util/simplify_expr_pointer.cpp

```diff
@@ -402,10 +392,7 @@ simplify_exprt::simplify_pointer_offset(const unary_exprt &expr)
        c_ptr.value_is_zero_string())
     {
       auto new_expr = from_integer(0, expr.type());
-
-      simplify_node(new_expr);
-
-      return new_expr;
+      return changed(simplify_node(new_expr));
```

Comment:
Is there anything to simplify here? AFAIK we don't usually run `simplify_node` on the result of `from_integer`.